lifecycle-operator/controllers/lifecycle/keptnapp/controller.go

```diff
@@ -54,6 +54,9 @@ type KeptnAppReconciler struct {
 // +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappversion,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappversion/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappversion/finalizers,verbs=update
+// +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappcontexts,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappcontexts/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=lifecycle.keptn.sh,resources=keptnappcontexts/finalizers,verbs=update
```

Comment:
i think these two lines can be removed, as the controller does not update the KeptnAppContext's status, but only reads its spec

Reply:
Alright.